Source/WebInspectorUI/UserInterface/Models/Resource.js

```diff
@@ -721,7 +723,7 @@ WI.Resource = class Resource extends WI.SourceCode
         return requestDataContentType && requestDataContentType.match(/^application\/x-www-form-urlencoded\s*(;.*)?$/i);
     }
 
-    updateForResponse(url, mimeType, type, responseHeaders, statusCode, statusText, elapsedTime, timingData, source, security)
+    updateForResponse(url, mimeType, type, responseHeaders, statusCode, statusText, integrity, elapsedTime, timingData, source, security)
```

Comment:
I'm getting closer to saying this should be a single destructured object each time you add something 😅 – definitely something for a separate patch though.